utilities/write_batch_with_index/write_batch_with_index_test.cc

```diff
@@ -1634,6 +1634,17 @@ TEST_F(WBWIOverwriteTest, MutateWhileIteratingBaseStressTest) {
   ASSERT_OK(iter->status());
 }
 
+TEST_P(WriteBatchWithIndexTest, TestNewIteratorWithBaseFromWbwi) {
+  ColumnFamilyHandleImplDummy cf1(6, BytewiseComparator());
+  KVMap map;
+  map["a"] = "aa";
+  map["c"] = "cc";
+  map["e"] = "ee";
+  std::unique_ptr<Iterator> iter(
+      batch_->NewIteratorWithBase(&cf1, new KVIter(&map)));
+  ASSERT_NE(nullptr, iter);
+}
```

Comment:
Please make sure the iter is valid (or not) and check the status to be ok (or not).  This likely will fail the ASSERT_STATUS_CHECKED tests otherwise.

Reply:
@mrambacher Good point! I just added a commit that addresses that. Thanks